### ISSUE TITLE
Method of exploiting factions

### ISSUE THREAD

**Issue by skidbld** [MassiveCraft/Factions#824]
People can protect their land from griefers by block-locking all of their chests, then logging off if someone attacks. Thanks to the offline mechanic, right now people can do this, with ease. Could you set an option so the offline mechanic only triggers when people have been offline for a certain amount of time?

**Comment by ulumulu1510** [comment]
Please provide us with more info:
MassiveCore version?
Factions version?
Bukkit/spigot version?

Would you please elaborate on the situation, what is happening exactly and what of it is related to factions?

Block-locking is not done by factions. It might be more effective to set up a rule against this on your server, with an appropriate punishment.

**Comment by skidbld** [comment]
MassiveCore 2.7.5, Factions 2.7.5. If you read my post correctly, I said that people could block-lock their chests to be invincible from all forms of raiding. Block locking a chest is just placing a block on top of it so others cannot open it. Factions 2.7.5 prevents explosions in a faction's land if that faction does not have any players online by default.

If people log off right when they are getting raided, their base becomes TNT proof, and thus their chests can also be invincible from raiders.

What I am asking for:

A set delay where people have to be offline for a certain amount of time before they get TNT protection.

What this will do:

Prevent people from exploiting the flag preventing explosions in their land when they are offline.

**Comment by ulumulu1510** [comment]
I now understand your problem, because you never mentioned tnt before. 

Personally, I see it like this: If they players want to lock themself in, they will eitherway. If we delay it, they'll react to the delay and time better, log off sooner. And if the delay is too long, your factions would be griefed in no time and the flag would miss its aim.

My conclusion to your problem would be to state in your server rules, that missusing plugin mechanics (such as you describe ) will result a ban/jail or land taken from them. This will be yome manual work, but'll also apply to other situations and protect you as a server owner from more harm.

TL;Dr: Make a rule that forbids it, the change suggested is obsolete with a good ruleset.

**Comment by magnusulf** [comment]
Duplicate of #365
